src/settings.ts

```diff
@@ -35,16 +35,13 @@ export const PRIVACY_POLICY = IS_BURN
   ? SPARKLEVERSE_PRIVACY_POLICY
   : SPARKLE_PRIVACY_POLICY;
 
+export const SPARKLE_ICON = "/sparkle-icon.png";
 export const DEFAULT_MAP_BACKGROUND = "/maps/Sparkle_Field_Background.jpg";
 export const DEFAULT_VENUE_BANNER = "/assets/Sparkle_Banner_Default.jpg";
 export const DEFAULT_VENUE_LOGO = "/assets/Sparkle_SquareLogo_Default.jpg";
 export const DEFAULT_PROFILE_IMAGE = "/anonymous-profile-icon.jpeg";
 export const DEFAULT_AVATAR_IMAGE = "/icons/sparkle-nav-logo.png";
 export const DEFAULT_PARTY_NAME = "Anon";
-export const SPARKLEVERSE_LOGO_URL = "/sparkleverse-logo.png";
```

Comment:
This still seems like it is relevant to `src/pages/Account/Step6b.tsx:28`

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1120/commits/b7fd02de0a031b10779e5e6c327d1caefe070aba